modules/planning/open_space/coarse_trajectory_generator/hybrid_a_star.cc

```diff
@@ -101,32 +82,27 @@ bool HybridAStar::RSPCheck(
     std::shared_ptr<Node3d> node = std::shared_ptr<Node3d>(new Node3d(
         reeds_shepp_to_end->x[i], reeds_shepp_to_end->y[i],
         reeds_shepp_to_end->phi[i], XYbounds_, planner_open_space_config_));
-    if (!ValidityCheck(node, obstacles_vertices_vec)) {
+    if (!ValidityCheck(node)) {
       return false;
     }
   }
   return true;
 }
 
-bool HybridAStar::ValidityCheck(
-    std::shared_ptr<Node3d> node,
-    const std::vector<std::vector<common::math::Vec2d>>&
-        obstacles_vertices_vec) {
-  if (obstacles_vertices_vec.size() == 0) {
+bool HybridAStar::ValidityCheck(std::shared_ptr<Node3d> node) {
+  if (obstacles_linesegments_vec_.size() == 0) {
     return true;
   }
-  for (const auto& obstacle_vertices : obstacles_vertices_vec) {
-    size_t vertices_num = obstacle_vertices.size();
-    for (size_t i = 0; i < vertices_num - 1; ++i) {
-      common::math::LineSegment2d line_segment = common::math::LineSegment2d(
-          obstacle_vertices[i], obstacle_vertices[i + 1]);
-      if (node->GetBoundingBox(vehicle_param_).HasOverlap(line_segment)) {
+  for (const auto& obstacle_linesegments : obstacles_linesegments_vec_) {
+    for (const common::math::LineSegment2d& linesegment :
+         obstacle_linesegments) {
+      if (node->GetBoundingBox(vehicle_param_).HasOverlap(linesegment)) {
         return false;
       }
     }
   }
   return true;
-}
+}  // namespace planning
```

Comment:
remove this

Reply:
remove what? the "// namespace planning"?

Reply:
Done